src/deb/helios-agent/helios-agent.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Helios agent daemon
```

Comment:
I think we also want to add a `After=network.target` to this section to signal that this service should be started after the network is available